### DISCUSSION TITLE
Overview for one car

### DISCUSSION THREAD

**Discussion by ednax** [pkuehnel/TeslaSolarCharger#1110] [Ideas]
Hi,

is it possible to select a single car to show in the overview?
Maybe something like http://<TSC-HOST>:7190/Overview?car_id=1
I have access to two teslas and want to show the TSC-Overview in the car - but only the overview for the car I'm currently in.
Is that possible or maybe coud you please add this feature?

thanks
Ednax

**Comment by pkuehnel** [comment]
Currently not possible, I add an issue for that.
